src/components/dock/examples/dock-notification.tsx

```diff
@@ -0,0 +1,96 @@
+import { Component, h, State } from '@stencil/core';
+import { DockItem } from '../dock.types';
+
+/**
+ * Displaying a notification badge
+ *
+ * It is possible to display a notification badge on each individual
+ * button in the Dock. Badges are supposed to inform the user that
+ * there is something in the menu that requires their attention.
+ *
+ * This is typically done by displaying a number, which summarizes
+ * the quantity of the items that require user's attention.
+ * However, it is also possible to use a short string such as "·" or "!"
+ * when this quantity is unclear or undefined.
+ *
+ * :::important
+ * The menus are not a part of the Dock. They are individual components
+ * that you develop separately. Make sure that the information
+ * and interactions regarding the notifications are correctly handled.
+ *
+ * For example, when the items that require user's attention are
+ * seen or handled by the user after opening the menu, the badge on the
+ * Dock button should disappear.
+ * :::
+ *
+ * :::warning
+ * Do not negatively exploit this possibility and spam users' awareness.
+ * The Dock is the most important and most dominant structural part of
+ * the UI of your application. Therefore crowding it with too much noise
+ * Will negatively affect the user experience.
```

Comment:
If you want to emphasize "will" here, I suggest instead 🙂 

```suggestion
 * _will_ negatively affect the user experience.
```